mozilla/activity-stream
# Bug 1562277 - Spoc dismiss flow updates.

To test:

1. Ensure your block list is empty.
2. Load a new tab with spocs.
3. Dismiss a spoc, it should be replaced with the next, non spoc card.
4. Refresh, a new, spoc, but not the one you dismissed should be back.
